Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -868,7 +868,7 @@ private struct FirefoxHomeHeaderViewUX {
     static let TextFont = DynamicFontHelper.defaultHelper.SmallSizeHeavyWeightAS
     static let ButtonFont = DynamicFontHelper.defaultHelper.MediumSizeBoldFontAS
     static let SeparatorHeight = 0.5
-    static let Insets: CGFloat = UIDevice.current.userInterfaceIdiom == .pad ? FirefoxHomeUX.SectionInsetsForIpad + FirefoxHomeUX.MinimumInsets : FirefoxHomeUX.MinimumInsets
+    static let Insets: CGFloat = UIDevice.current.isPad ? FirefoxHomeUX.SectionInsetsForIpad + FirefoxHomeUX.MinimumInsets : FirefoxHomeUX.MinimumInsets
```

Comment:
I suggest replacing this with a UITraitCollection based approach. You could be on an iPad, but in split screen mode.